src/simulation/dynamics/Thrusters/thrusterDynamicEffector/thrusterDynamicEffector.cpp

```diff
@@ -184,7 +184,7 @@ void ThrusterDynamicEffector::ConfigureThrustRequests(double currentTime)
 void ThrusterDynamicEffector::UpdateThrusterProperties()
 {
     // Save hub variables
-    Eigen::Vector3d r_BN_N = this->hubPosition->getState();
+    Eigen::Vector3d r_BN_N = (Eigen::Vector3d)*this->inertialPositionProperty;
```

Comment:
Just a comment: it might be worth us looking into (separate from this PR) what refactoring we could do on the dynParamManager to provide strongly typed assessors for this so that we can remove this unfortunate type casting.